ctapipe/tools/camdemo.py

```diff
@@ -18,12 +18,12 @@
 
 
 class CameraDemo(Tool):
-    name = "ctapipe-camdemo"
+    name = "ctapipe.camdemo"
```

Comment:
Why this change?

Reply:
I hope my memory does not let me down, but this was needed for more setting different levels in the config (-file?). eg
one "global" setting for all of `ctapipe`, a different  one for `ctapipe.io` .. this way `ctapipe.camdemo` would get the gloabl ctapipe setting

Reply:
But this name is what the tool is called on the command line, there should be no dot here.

We should find a different solution. Even if this was the correct solution, we would need to change that in every tool.

Reply:
Also, when the components are correctly setup, everything should start from this top level tool.